veocean-data-farming/df-background.md

```diff
@@ -0,0 +1,55 @@
+---
+description: Intuition and background on the Data Farming program.
+---
+# Data Farming Background
+
+It rewards OCEAN to liquidity providers (stakers) as a function of consume volume and liquidity. It’s like DeFi liquidity mining, but tuned for data consumption. DF’s aim is to achieve a minimum supply of data for network effects to kick in, and once the network flywheel is spinning, to increase growth rate.
```

Comment:
It might be clearer just to describe them as stakers. People might assume that we're running an AMM if we use terms like liquidity providers

Reply:
Yes, agree. We shouldn't use "liquidity provider" anywhere, it's inaccurate.

Reply:
Yeah, I agree. Done.